apps/storybook/src/Tile.stories.tsx

```diff
@@ -41,97 +40,109 @@ export default {
   title: 'Core/Tile',
 } as Meta<TileProps>;
 
-export const Basic: Story<TileProps> = (props) => {
-  const {
-    name,
-    description,
-    metadata,
-    badge,
-    thumbnail,
-    leftIcon,
-    rightIcon,
-    moreOptions,
-    ...rest
-  } = props;
+export const Basic: Story<TileProps> = (args) => {
   return (
-    <Tile
-      name={name}
-      description={description}
-      metadata={metadata}
-      badge={badge}
-      thumbnail={thumbnail}
-      leftIcon={leftIcon}
-      rightIcon={rightIcon}
-      moreOptions={moreOptions}
-      {...rest}
-    />
+    <Tile {...args}>
+      <Tile.Name>
+        <Tile.NameIcon />
+        <Tile.NameLabel>Stadium</Tile.NameLabel>
+      </Tile.Name>
```

Comment:
when the label is a simple string and when there is no custom icon, we can just show `<Tile.Name name='Stadium'>`

Reply:
I think for Storybook it is better to keep it this way so that users can toggle the icons, since if only `name` prop is passed in, there will be no icons, and if there is NameIcon subcomponent, it overrides the `name` (I talked about this in another comment below)

Reply:
can we change the website examples and at least the basic story?

Reply:
yea makes sense. I updated this.